locale/en/blog/release/v6.9.0.md

```diff
@@ -0,0 +1,145 @@
+version: 6.9.0
+category: release
+title: Node v6.9.0 (LTS)
+slug: node-v6-9-0
+layout: blog-post.hbs
+author: Rod Vagg
+---
+
+_This is a security release. All Node.js users should consult the security release summary at https://nodejs.org/en/blog/vulnerability/october-2016-security-releases/ for details on patched vulnerabilities._
+
+### A New LTS Release Line
+
+v6.9.0 marks the transition of Node.js v6 into Long Term Support (LTS) with the codename **"Boron"**. The v6 release line now moves in to "Active LTS" and will remain so until April 2018. After that time it will move in to "Maintenance" until end of life in April 2019.
+
+### LTS Migration
+
+Some highlights for users migrating from Node.js v4 LTS "Argon" to Node.js v6 LTS "Boron":
+
+* The `Buffer()` constructor has been deprecated in the documentation in favour of the new `Buffer.from()`, `Buffer.alloc()` and `Buffer.allocUnsafe()` for security and safety. See the [Buffer documentation](https://nodejs.org/api/buffer.html) for full details. As a documentation-only deprecation there will be no warnings printed to standard error, this is the first phase in a long deprecation cycle.
+* Support has been dropped for Windows Vista and earlier and macOS 10.7 and earlier.
+* Many warning messages and error messages have been cleaned up and made more consistent.
+* The [`--prof-process`](https://nodejs.org/api/cli.html#cli_prof_process) command line argument can be used to process output files created when using the V8 `--prof` command line argument.
+* A new [`EventEmitter#eventNames()`](https://nodejs.org/api/events.html#events_emitter_eventnames) method can be used to list all events currently being listened to on an `EventEmitter`.
+* [`fs.mkdtemp()`](https://nodejs.org/api/fs.html#fs_fs_mkdtemp_prefix_options_callback) is a fast and safe way to make a unique temporary directory using operating system primitives.
+* [`process.cpuUsage()`](https://nodejs.org/api/process.html#process_process_cpuusage_previousvalue) will allow insight into CPU resources being consumed by the current process.
+* Very large arrays are now truncated when passed through `util.inspect()`, this also applies to `console.log()` and friends.
+* When a native `Promise` incurs a rejection but there is no handler to receive it, a warning will be printed to standard error.
+* A new experimental debugging protocol can be activated with the `--inspector` command line argument. This uses the "v8_inspector" protocol and can be consumed directly by Chrome DevTools, Visual Studio Code and others.
+
+While there are some breaking API changes appearing in both [v5.0.0](https://direct.nodejs.org/en/blog/release/v5.0.0/) and [v6.0.0](https://direct.nodejs.org/en/blog/release/v6.0.0/), they are relatively minor and should not have significant impact on most users.
+
+In addition to Node.js changes, we also have a number of V8 upgrades between Node.js v4 and v6.9.0, including these JavaScript language enhancements:
```

Comment:
Should this be `Node.js v4 LTS and v6.9.0` or `Node.js v4 LTS and the release of the v6 LTS line?`